features/core_min_bootstrap.feature

```diff
@@ -66,44 +66,44 @@ Feature: register a salt-minion via bootstrap
     And I click on "Update Activation Key"
     Then I should see a "Activation key Minion testing has been modified" text
 
-  Scenario: bootstrap a sles minion with an activation-key
+  Scenario: Bootstrap a SLES minion with an activation key
      Given I am authorized
-     And I go to the bootstrapping page
+     When I go to the bootstrapping page
      Then I should see a "Bootstrap Minions" text
-     And  I enter the hostname of "sle-minion" as hostname
+     When I enter the hostname of "sle-minion" as hostname
      And I enter "22" as "port"
      And I enter "root" as "user"
      And I enter "linux" as "password"
      And I select "1-MINION-TEST" from "activationKeys"
      And I click on "Bootstrap"
-     Then I wait until i see "Successfully bootstrapped host! " text
+     Then I wait until I see "Successfully bootstrapped host! " text
      And I navigate to "rhn/systems/Overview.do" page
      And I wait until I see the name of "sle-minion", refreshing the page
      And I wait until onboarding is completed for "sle-minion"
 
-  Scenario: verify minion bootstrapped with activation key, packages
+  Scenario: Verify that minion bootstrapped with Salt key and packages
```

Comment:
good catch then :)

Reply:
Yeah, from steps below it was not an activation key but a Salt key